### ISSUE TITLE
Scrollable element is not keyboard accessible - Crime/Fire log

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9366]
# Motivation

SiteImprove is reporting an accessibility issue, scrollable element is not keyboard accessible. This is reported as a wcag 2.1 A conformance issue.

This page has a scrollable section that can't be operated by keyboard.

# Additional Info

Crime Log
https://my2.siteimprove.com/Inspector/1179985/A11y/Page?pageId=55442230690&impmd=264ED37B65FA6186129334A508391970&conf=a#/sia-r84/failed/U2hORlpsbDc=;NDY2YjUwMzg=

Fire Log
https://my2.siteimprove.com/Inspector/1179985/A11y/Page?pageId=55442231381&impmd=264ED37B65FA6186129334A508391970&conf=a#/sia-r84/failed/R3pGTVFPWU8=;ZWVjMmQwMTk=
